### PR TITLE
Remove HDInsight related common error messages

### PR DESCRIPTION
**Pull request by wezhang**

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/cosmosserverlessspark/spark/ui/livy/batch/KillCosmosServerlessSparkBatchJobAction.kt [hunk 1]
```diff
@@ -41,44 +41,48 @@ import java.io.IOException
...
-                        PluginUtil.displayErrorDialog("Kill Serverless Spark Job", errMsg)
-                        log().warn("Failed to kill serverless spark job. " + ExceptionUtils.getStackTrace(err))
-                        EventUtil.logErrorWithComplete(operation, ErrorType.serviceError, IOException(errMsg), null, null)
-                    }
-                )
+                        )
+            }
+            return false
+        } catch (ignored: RuntimeException) {
+            return true
```
Comment (t-rufang) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/cosmosserverlessspark/spark/ui/livy/batch/KillCosmosServerlessSparkBatchJobAction.kt:85]: do we need to log the exception locally in logs?

File: Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/serverexplore/hdinsightnode/StorageAccountNode.java [hunk 2]
```diff
@@ -62,10 +62,14 @@ protected void loadAdditionalActions() {
         if (clusterDetail instanceof ClusterDetail) {
             addAction("Open Storage in Azure Management Portal", new NodeActionListener() {
                 @Override
-                protected void actionPerformed(NodeActionEvent e) throws AzureCmdException {
+                protected void actionPerformed(NodeActionEvent e) {
                     String subscriptionId = clusterDetail.getSubscription().getSubscriptionId();
                     String storageRelativePath = ((ClusterDetail) clusterDetail).getId() + REST_SEGMENT_STORAGE_ACCOUNT;
-                    openResourcesInPortal(subscriptionId, storageRelativePath);
+                    try {
```
Comment (t-rufang) [inline Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/serverexplore/hdinsightnode/StorageAccountNode.java:68]: Why not put `try` after line 65

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/cosmosserverlessspark/spark/ui/livy/batch/KillCosmosServerlessSparkBatchJobAction.kt (hunk 1 above)
Comment (wezhang) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/cosmosserverlessspark/spark/ui/livy/batch/KillCosmosServerlessSparkBatchJobAction.kt:85]: We have logged the exceptions in code. So, needn't here.

File: Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/serverexplore/hdinsightnode/StorageAccountNode.java (hunk 2 above)
Comment (wezhang) [reply Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/serverexplore/hdinsightnode/StorageAccountNode.java:68]: Only `openResourcesInPortal` throws `AzureCmdException`, refer to https://github.com/microsoft/azure-tools-for-java/blob/6d10763058e4b6c2767f3ca4ff3dfb019c3ac9ab/Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/Node.java#L403 please.

**Pull request merged**